doc/kubernetes/modules/ROOT/pages/openshift.adoc

```diff
@@ -0,0 +1,57 @@
+= Deploying on OpenShift
+:description: Deploy Keycloak on OpenShift with a setup similar to the minikube environment.
+
+{description}
+
+While minikube is the primary target for this setup, this tracks the progress on how to use it with an OpenShift environment, see <<supported-functionality>>.
+
+WARNING: OpenShift must not be accessible from the Internet, as this setup opens node ports and URLs that are not properly secured by good passwords and transport encryption.
+
+== Prerequisites
+
+* OpenShift with admin rights for a user to be able to install custom resource definitions
```

Comment:
do we want to point to some docs to establishing connection to a remote openshift cluster, such as login vs kubeconfig and is there any preferred way of doing that ?

Reply:
@kami619 - it would be good to have docs and maybe pointers to additional docs for that. Please create a new issue for this so we can tackle this after this PR has been merged.

Reply:
wfm, I will create the issue. thanks Alex.